lib/secrets/secrets.go

```diff
@@ -156,18 +179,37 @@ func (d *diskStore) Exists(secret SecretName) (bool, error) {
 }
 
 func (d *diskStore) GetSecret(secret SecretName) ([]byte, error) {
-	path := filepath.Join(d.basePath, secret.Group, secret.Name)
-	ret, err := ioutil.ReadFile(path)
+	var ret []byte
```

Comment:
I don't think ret needs to be declared here. It gets unconditionally set outside the following if block

Reply:
removed now